### PR TITLE
Add official Backdrop docs

### PR DESCRIPTION
**Pull request by pirog**
This goes along with https://github.com/docker-library/official-images/pull/1584

### CODE REVIEW

**Comment by jenlampton** [comment]
Travis :(

**Comment by pirog** [comment]
i guess docker's markdown doesn't love my default "treat tabs as spaces" drupal code guidelines. Should be good now i think.

**Comment by jenlampton** [comment]
:+1:

File: backdrop/content.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+The following Docker Hub features can help with the task of keeping your dependent images up-to-date:
+
+-	[Automated Builds](https://docs.docker.com/docker-hub/builds/) let Docker Hub automatically build your Dockerfile each time you push changes to it.
+-	[Repository Links](https://docs.docker.com/docker-hub/builds/#repository-links) can ensure that your image is also rebuilt any time `backdrop` is updated.
+
+# License
+
+View [license information](https://backdropcms.org/license) for the software contained in this image.
+
+# Supported Docker versions
```
Comment (yosifkit) [inline backdrop/content.md]: This section and the "User Feedback" sections are generated by the update script.  If you want a custom "User Feedback" section not based upon [.template-helpers/user-feedback.md](https://github.com/docker-library/docs/blob/09413030581c4f7239dad4602776fbf6b335a56c/.template-helpers/user-feedback.md), then just put it in a `user-feedback.md`in this `backdrop/` folder.

**Comment by pirog** [comment]
@jenlampton i think its def best for you to make the call on @yosifkit's comments here :)

**Comment by jenlampton** [comment]
Ah, I didn't realize that was boilerplate. Feel free to remove my changes
to the user feedback section .

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

**Comment by pirog** [comment]
@yosifkit removed user feedback section

**Comment by yosifkit** [comment]
Thanks @pirog! Can you also remove the "Supported Docker versions" section; it too is generated.

**Comment by pirog** [comment]
Battle of attrition here i guess :) 

Yes

**Comment by pirog** [comment]
ALRIGHT @yosifkit i think we should be good!!

**Comment by yosifkit** [comment]
LGTM, squash merged via 8bd485877ec9edc2ff2b7b495816053621cb5c43

**Pull request closed without merging**